Library/Formula/prettyping.rb

```diff
@@ -0,0 +1,11 @@
+require 'formula'
+
+class Prettyping < ScriptFileFormula
+  homepage 'http://denilsonsa.github.io/prettyping/'
+  url 'https://raw.githubusercontent.com/denilsonsa/prettyping/v1.0.0/prettyping'
+  sha1 '5e0296f9f28834d0985bb9f3452e777b35291350'
+  version '1.0.0'
+  def install
+    bin.install 'prettyping'
+  end
+end
```

Comment:
Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!